File: Web.Admin/ClientApp/src/app/models/address/AddressCreateParameters.ts
export class AddressCreateParameters {

  organizationID: number;
  orderIndex: number;
  street1: string;
  street2: string;
  street3: string;
  locality: string;
  region: string;
  country: string;
  postalCode: string;

  public constructor(organizationID: number,
    orderIndex: number,
    street1: string,
    street2: string,
    street3: string,
    locality: string,
    region: string,
    country: string,
    postalCode: string) {
    this.organizationID = organizationID;
    this.orderIndex = orderIndex;
    this.street1 = street1;
    this.street2 = street2;
    this.street3 = street3;
    this.locality = locality;
    this.region = region;
    this.country = country;
    this.postalCode = postalCode;
  };

}

File: Libraries/Internal/NIST.CVP.Libraries.Internal.ACVPCore/Models/Parameters/AddressCreateParameters.cs
namespace NIST.CVP.Libraries.Internal.ACVPCore.Models.Parameters
{
	public class AddressCreateParameters
	{
		public long OrganizationID { get; set; }
		public int OrderIndex { get; set; }
		public string Street1 { get; set; }
		public string Street2 { get; set; }
		public string Street3 { get; set; }
		public string Locality { get; set; }
		public string Region { get; set; }
		public string Country { get; set; }
		public string PostalCode { get; set; }
	}
}
